src/kohlrahbi/scrape_pruefis/collect_pruefis.py

```diff
@@ -0,0 +1,76 @@
+from datetime import date
+from pathlib import Path
+
+import click
+import docx  # type:ignore[import]
+import tomlkit
+from docx.table import Table  # type:ignore[import]
+from maus.edifact import EdifactFormatVersion
+
+from kohlrahbi.docxfilefinder import DocxFileFinder
+from kohlrahbi.logger import logger
+from kohlrahbi.read_functions import does_the_table_contain_pruefidentifikatoren, get_all_paragraphs_and_tables
+from kohlrahbi.seed import Seed
+
+
+@click.command()
+@click.option(
+    "--format-version",
+    multiple=True,
+    default=[e.value for e in EdifactFormatVersion],  # Set default to all known format versions
+    show_default="All",
+    type=click.Choice([e.value for e in EdifactFormatVersion], case_sensitive=False),
+    help="Format version(s) of the AHB documents. Default is all known format versions.",
+)
+def update_pruefis(format_version):
+    """
+    This CLI tool updates the all_known_pruefis.toml files with Prüfidentifikatoren from AHB documents.
+    If no specific format version is provided, it processes all known format versions.
+    """
+    for version in format_version:  # Iterate over each provided format version
+        all_pruefis: dict[str, str] = {}
+
+        edi_energy_mirror_repo_root_path = Path(__file__).parents[4] / "edi_energy_mirror"
```

Comment:
klappt das immer? oder sollte man die möglichkeit haben, das von außen zu überschreiben? also wenn ich z.b. kohlrahbi installiere liegt der doch in den site-packags

Reply:
fair point. Das war damals auch hart hinterlegt. Ich bau es mal ein dass man es auch angeben kann